cisco_umbrella_dns/README.md

```diff
@@ -0,0 +1,53 @@
+# Cisco Umbrella DNS Integration for Datadog
+
+## Overview
+
+Cisco Umbrella is the leading platform for network DNS security monitoring. Umbrella's DNS-layer security offers a fast and easy way to enhance security, providing improved visibility and protection for users both on and off the network. By preventing threats over any port or protocol before they reach the network or endpoints, Umbrella DNS-layer security aims to deliver the most secure, reliable, and fastest internet experience to over 100 million users.
+
+This integration offers visualizations and log enrichments for Cisco Umbrella DNS and Proxy logs.
```

Comment:
little nit here:

Its unclear what log enrichment means. I doubt this is using actual enrichment via a lookup processor. Does this mean theres is a pipeline? or are there dashboards populated by logs?

Should also include, at a high level: 

what are some examples of logs ? 
why would users want to ingest them into the platform? 
what benefits do they get from provided visualizations?

Reply:
+1

Reply:
Updated the README, please have a look.